core/services/standardcapability/standard_capability.go

```diff
@@ -0,0 +1,93 @@
+package standardcapability
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/loop"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/job"
+	"github.com/smartcontractkit/chainlink/v2/plugins"
+)
+
+type StandardCapability struct {
+	log                logger.Logger
+	spec               *job.StandardCapabilitySpec
+	pluginRegistrar    plugins.RegistrarConfig
+	telemetryService   core.TelemetryService
+	store              core.KeyValueStore
+	capabilityRegistry core.CapabilitiesRegistry
+	errorLog           core.ErrorLog
+	pipelineRunner     core.PipelineRunnerService
+	relayerSet         core.RelayerSet
+
+	capabilityLoop *loop.StandardCapabilityService
+}
+
+func NewStandardCapability(log logger.Logger, spec *job.StandardCapabilitySpec,
+	pluginRegistrar plugins.RegistrarConfig,
+	telemetryService core.TelemetryService,
+	store core.KeyValueStore,
+	capabilityRegistry core.CapabilitiesRegistry,
+	errorLog core.ErrorLog,
+	pipelineRunner core.PipelineRunnerService,
+	relayerSet core.RelayerSet) *StandardCapability {
+	return &StandardCapability{
+		log:                log,
+		spec:               spec,
+		pluginRegistrar:    pluginRegistrar,
+		telemetryService:   telemetryService,
+		store:              store,
+		capabilityRegistry: capabilityRegistry,
+		errorLog:           errorLog,
+		pipelineRunner:     pipelineRunner,
+		relayerSet:         relayerSet,
+	}
+}
+
+func (s *StandardCapability) Start(ctx context.Context) error {
```

Comment:
@ettec You might want to wrap this in a StartOnce helper: I'm not sure all of these functions below will behave nicely if this is called multiple times